### PR TITLE
Python article V1

### PR DESCRIPTION
**Pull request by ArneTR**
Hello Mr. Geerd-Dietger Hoffmann,

would you please be so kind and review this article. I wrote it myself.

I thank you very much.

Yours sincerly,

Mr. Arne Tarara from Berlin

### CODE REVIEW

**Comment by ArneTR** [comment]
Link: https://python-article.website-jic.pages.dev/case-studies/energy-efficiency-python/

File: content/case-studies/energy-efficiency-python.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+Since the [archived repository on Gitlab](https://salsa.debian.org/benchmarksgame-team/archive-alioth-benchmarksgame) contains mutliple submitted variants we resorted to using the originally selected benchmarks by the study authors in their [Github repository](https://github.com/greensoftwarelab/Energy-Languages)
+
+The [Green Metrics Tool](https://www.green-coding.io/projects/green-metrics-tool/) makes it very easy to consume these benchmarks directly. We only added a container which contained the necessary Python version:
+
+- [Python3.6 Dockerfile](https://github.com/green-coding-solutions/python-benchmarks/blob/main/python/Dockerfile-3-6)
+- [Python3.9 Dockerfile](https://github.com/green-coding-solutions/python-benchmarks/blob/main/python/Dockerfile-3-9)
+- [Python 3.12 Dockerfile](https://github.com/green-coding-solutions/python-benchmarks/blob/main/python/Dockerfile-3-12)
+- [Mojo Dockerfile](https://github.com/green-coding-solutions/python-benchmarks/blob/main/python/Dockerfile-Mojo)
+- [PyPy 3.10](https://github.com/green-coding-solutions/python-benchmarks/blob/main/python/Dockerfile-pypy)
```
Comment (ribalba) [inline content/case-studies/energy-efficiency-python.md:115]: You say 3.7 below

File: content/case-studies/energy-efficiency-python.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+
+
+
+
+    - What => Change to 3.6 is [significant](https://metrics.green-coding.io/compare.html?ids=4187414a-32ea-4bb4-b5d3-17496e307109,eff016f1-8bcd-4bc8-b1bb-f71da5296856,42d34b4a-d028-4d61-b2fe-aef3927035ec,e9e955f2-ae74-4904-8c25-70f0364fb780,45c0d2ed-97ef-4a88-8a2f-f5a8076f7ecc,90bdde06-e427-4986-9268-42ba91bc8fcc,bced3fe2-37e7-4823-88f7-dad14fe4367d,764bbf13-e5bf-4d32-8a15-21a92610b5fb,388d786b-66fc-4f52-af24-c620586d2eb5,1f85d489-c51e-4a0c-9e2c-d1f58be13028,2b6fa23f-db48-4cb3-8fb3-64ac73c4540e,c641bd55-13f5-40da-8c0f-3546fab8f69e,1d80f5d6-3978-4162-887c-363eca556814,f007e7a2-4e52-418e-871a-ab073b71c885,296cbaaa-972f-4a09-bb6a-57a37fc935c5,26c36b5e-0f82-4d0c-a0eb-78c9f3e4f686,3b026236-02d0-48d8-b2ef-bc1ebcd09133,de113e65-3bbf-47aa-a8a6-fd89cda20385,3cb69b09-bff0-4846-b64b-0a4e7f22c5a1,c21ff568-83ae-49b9-b4b4-57b35e8c3fda)! 
+
+
+
+## DISCUSSION
+
```
Comment (ribalba) [inline content/case-studies/energy-efficiency-python.md:210]: I would probably want a table here that shows the total energy values for each versions. As the tables above mean a lot of scrawling to compare.

File: content/case-studies/energy-efficiency-python.md (hunk 1 above)
Comment (ArneTR) [reply content/case-studies/energy-efficiency-python.md:115]: doh. it's 3.10

**Pull request merged**